src/librustdoc/lib.rs

```diff
@@ -43,6 +43,8 @@ extern crate rustc_typeck;
 extern crate test as testing;
 #[macro_use]
 extern crate log;
+#[cfg(windows)]
+extern crate termcolor;
```

Comment:
I'm a bit confused why this is here. Is there some reason the code can't directly reference the `termcolor` crate?

Reply:
```suggestion
```

Reply:
Hmm, I tried removing the existing `extern crate`s and got lots of errors about the crates not existing:
```
error[E0433]: failed to resolve: use of undeclared type or module `rustc_session`
  --> src/librustdoc/lib.rs:28:5
   |
28 | use rustc_session::config::{make_crate_type_option, ErrorOutputType, RustcOptGroup};
   |     ^^^^^^^^^^^^^ use of undeclared type or module `rustc_session`
```
Maybe there's something weird going on with bootstrap?

Reply:
The `rustc_*` crates must use `extern crate` because they are loaded implicitly from the sysroot (this is similar to how `extern crate test;` is still needed).  Dependencies listed in `Cargo.toml` do not need `extern crate`.